### DISCUSSION TITLE
Introducing helping dictionnary with acronyms, people names, place names...

### DISCUSSION THREAD

**Discussion by g-vidal** [openai/whisper#1268] [Ideas]
Thanks for an amazing tool and all the energy deployed to progress !
I have observed that  some errors are due to wrong understanding of acronyms or wrong spelling of  names. In many cases whisper recovers immediately and efficiently but in few cases it introduces a wrong verb or another unwanted word that perturbates the sentence.

It would be very usefull to have the opportunity to provide a short list of words and acronyms that may appear in the audio then give them a strong probability to help whisper decide properly when in doubt and  also provide a correct spelling  of names  which is few thing but with strong effect on readers. 
I hope I have not missed an existing solution somewhere in the whisper universe, if it is the case please forgive me and provide a link.

**Comment by glangford** [comment]
See these discussions:
* https://github.com/openai/whisper/discussions/963
* https://github.com/openai/whisper/discussions/328
* https://github.com/openai/whisper/discussions/117#discussioncomment-3727051

**Reply by g-vidal** [reply]
Thanks @glangford for not having  flammed the stupid newcomer  and for the very usefull answer !
In a first step I'll follow @jongwook 's  recommendation with `--initial_prompt`  from #963 and see if it is sufficient.
